parcels/scripts/utils.py

```diff
@@ -0,0 +1,120 @@
+import numpy as np
+import xarray as xr
+from os import path
+from netCDF4 import Dataset
+
+
+def compute_curvilinear_rotation_angles(mesh_filename, rotation_angles_filename, variables, dimensions):
```

Comment:
rename this file to `compute_curvilinear_rotation_angles.py`, just like the other files in the `scripts` directory?